.chloggen/default_limits_for_init_containers.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: set default resource limits for instrumentation init containers
```

Comment:
`Set default resource requirements for instrumentation init containers.`

Reply:
```suggestion
note: Set default resource requirements and limits for instrumentation init containers.
```